src/reason_pprint_ast.ml

```diff
@@ -3672,7 +3677,8 @@ class printer  ()= object(self:'self)
       | (lbl, expression) :: tail ->
          let nextAttr =
            match expression.pexp_desc with
-           | Pexp_ident (ident) when (Longident.last ident.txt) = lbl -> atom lbl
+           | Pexp_ident (ident) when ((isLongIdentWithDot ident.txt) == false 
```

Comment:
Hmm, this is an artefact from https://github.com/facebook/reason/pull/1102. I did branch from that PR. Not sure why this is still in here, I thought I rebased 🤔